hudi-client/hudi-client-common/src/main/java/org/apache/hudi/table/marker/DirectWriteMarkers.java

```diff
@@ -130,6 +132,53 @@ public Set<String> createdAndMergedDataPaths(HoodieEngineContext context, int pa
     return dataFiles;
   }
 
+  public Set<String> appendedLogPaths(HoodieEngineContext context, int parallelism) throws IOException {
```

Comment:
Currently createdAndMergedDataPaths not return any log file markers because the change to generate 'create' type marker for log file has been reverted. I notice you also comment the code about this change. Let me explain it there.